pages/linux/konsave.md

```diff
@@ -0,0 +1,32 @@
+# konsave
+
+> A CLI program that lets you save and apply your Linux customizations with just one command!
+> More information: <https://github.com/Prayag2/konsave>.
+
+- Save the current configuration as a profile:
+
+`konsave --save {{profile_name}}`
+
+- Apply a profile:
+
+`konsave --apply {{profile_name}}`
+
+- Save the current configuration as a profile, overwriting existing profiles if they exist with the same name:
+
+`konsave -s {{profile_name}} --force`
+
+- List all profiles:
+
+`konsave --list`
+
+- Remove a profile:
+
+`konsave --remove {{profile_name}}`
+
+- Export a profile as a `.knsv` to the home directory:
+
+`konsave --export-profile {{profile_name}}`
+
+- Import a `.knsv` profile:
+
+`konsave --import-profile {{path to the file}}`
```

Comment:
I'm not sure if it should be `profile_name.knsv`. Feel free to replace it base on your knowledge.
```suggestion
`konsave --import-profile {{path/to/profile_name.knsv}}`
```

Reply:
Why not `profile.knsv`?

Reply:
Because all other examples use `profile_name` as the placeholder.

Reply:
So Should I Change it ?

Reply:
> So Should I Change it ?

Yeah